### PR TITLE
Exercise for new contributor

### PR DESCRIPTION
**Pull request by Parimal7**
Completed the exercise for new contributors as mentioned [here](https://gcompris.net/wiki/An_exercise_for_new_contributors).

### CODE REVIEW

File: src/activities/parimal/ActivityInfo.qml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2018 Your Name <yy@zz.org>
```
Comment (petitlapin) [inline src/activities/parimal/ActivityInfo.qml]: should be updated

File: src/activities/parimal/ActivityInfo.qml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import GCompris 1.0
+
+ActivityInfo {
+  name: "parimal/Parimal.qml"
+  difficulty: 1
+  icon: "parimal/parimal.svg"
+  author: "Parimal Prasoon &lt;parimalprasoon7@gmail.com&gt;"
+  demo: true
+  //: Activity title
+  title: "Parimal activity"
```
Comment (petitlapin) [inline src/activities/parimal/ActivityInfo.qml]: don't forget qsTr() for strings that are translatable

File: src/activities/parimal/Parimal.qml [hunk 3]
```diff
@@ -0,0 +1,134 @@
+/* GCompris - parimal.qml
+ *
+ * Copyright (C) 2018 PARIMAL PRASOON <parimalprasoon7@gmail.com>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
```
Comment (petitlapin) [inline src/activities/parimal/Parimal.qml]: if it's not a port, it's not needed else you can replace it with the original author

File: src/activities/parimal/Parimal.qml [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            font.pixelSize: 50
+        }
+
+
+        property bool running: false
+
+        Rectangle{
+            id: object2
+            anchors.centerIn: parent
+            height: 500
```
Comment (petitlapin) [inline src/activities/parimal/Parimal.qml]: we need to be careful when setting dimensions, as we target multiple devices (phones, tablets, computers) and screen resolutions (not necessary fullscreen).

File: src/activities/parimal/Parimal.qml [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+            width: 500
+            color: "#f56a6a"
+            RotationAnimation on rotation {
+            from: 0
```
Comment (petitlapin) [inline src/activities/parimal/Parimal.qml]: indentation

File: src/activities/parimal/Parimal.qml [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+            to: -360
+            duration: 3000
+            running: background.running
+            loops: Animation.Infinite
+
```
Comment (petitlapin) [inline src/activities/parimal/Parimal.qml]: no need to have multiple line breaks

**Comment by Parimal7** [comment]
Thanks for the review, I will keep the comments in mind.

**Pull request closed without merging**